packages/@react-stately/virtualizer/src/tween.ts

```diff
@@ -42,7 +36,7 @@ export function tween(begin, end, duration, ease, fn): CancelablePromise<void> {
     raf_id = requestAnimationFrame(function run(t) {
       // if we're using a high res timer, make sure timestamp is not the old epoch-based value.
       // http://updates.html5rocks.com/2012/05/requestAnimationFrame-API-now-with-sub-millisecond-precision
-      if (fixTs) {
+      if (t < 1e12) {
```

Comment:
Actually, I think this is the opposite of what we need? If the time is less than 1e12 then it's a high res timer already. I think we want `t > 1e12` here instead?

Reply:
And also we might need to check the resolution of `performance.now()` as the above check does. e.g. in safari `performance.now()` is less than 1e12 (I think after the spectre/meltdown attacks).

Maybe we should just put the check above in an if statement to check if `requestAnimationFrame` even exists?

Reply:
I will experiment with this and get back to you!

Reply:
Finished testing, doing the compare inside the animation seems to be the only option :(
I have updated the PR + added some `const`s to ease my conscience 😁